tasks/nunjucks.js

```diff
@@ -22,6 +22,9 @@ module.exports = function(grunt) {
         }
 
         var envOptions = { watch: false };
+        if (typeof options.autoescape === 'boolean') {
+            envOptions.autoescape = options.autoescape;
+        }
```

Comment:
@ArmorDarks I think it could be something like this:

```js
var envOptions = {
        watch: false,
        autoescape: options.autoescape || true
};
```